scala/ppml/demo/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM intelanalytics/bigdl-ppml-trusted-big-data-ml-python-graphene:2.0.0
 
-RUN NIGHTLY_VERSION=$(echo $(echo `wget -qO - https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/bigdl/bigdl-ppml-spark_$SPARK_VERSION/$BIGDL_VERSION/maven-metadata.xml \
+RUN NIGHTLY_VERSION=$(echo $(echo `wget -qO - https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/bigdl/bigdl-ppml-spark_3.1.2/2.0.0-SNAPSHOT/maven-metadata.xml \
```

Comment:
Do not hard code version in dockerfile or script unless it's necessary

Reply:
Update in this commit https://github.com/intel-analytics/BigDL/pull/4237/commits/596d408613078731269dd6983698c1eff9a675a0.